README.md

```diff
@@ -22,6 +23,8 @@ If you have OpenSSL installed in a non-standard directory, you can pass the loca
 % PKG_CONFIG_PATH=/path/to/openssl/lib/pkgconfig cmake .
 ```
 
+You need to delete `CMakeCache.txt` when you get an error [Undefined symbols for architecture x86_64: "_SSL_is_server"](https://github.com/h2o/h2o/issues/2093).
```

Comment:
If this is the concern, I think we should switch to describing off-tree build (i.e. create a separate directory for building, and build there) in the steps above.

Also, I think that this type of advice should go into H2O wiki (i.e. h2o/h2o/wiki), as it is common to all the projects.